core/cmd/ocr2vrf_configure_commands.go

```diff
@@ -261,6 +264,10 @@ func (cli *Client) ConfigureOCR2VRFNode(c *clipkg.Context) (*SetupOCR2VRFNodePay
 	} else {
 		err = fmt.Errorf("unknown job type: %s", c.String("job-type"))
 	}
+
+	// Close application.
+	app.Stop()
```

Comment:
Would it make sense to defer this when the app is started?

Reply:
> My bad!

This has been a lurking issue for us, had to happen sooner or later!